core/src/nucleus/ribosome/api/meta.rs

```diff
@@ -0,0 +1,68 @@
+use crate::{
+    nucleus::ribosome::{api::ZomeApiResult, Runtime},
+};
+use holochain_wasm_utils::api_serialization::meta::{MetaArgs,MetaMethod,MetaResult};
+use wasmi::{RuntimeArgs, RuntimeValue};
+use holochain_core_types::GIT_HASH;
+use holochain_core_types::hdk_version::HDK_VERSION;
+use std::convert::TryFrom;
+
+/// ZomeApiFunction::Meta function code
+/// args: [0] encoded MemoryAllocation as u64
+/// Expecting a string as complex input argument
+/// Returns an HcApiReturnCode as I64
+pub fn invoke_meta(runtime: &mut Runtime, args: &RuntimeArgs) -> ZomeApiResult {
+    let context = runtime.context()?;
+
+    let args_str = runtime.load_json_string_from_args(&args);
+    let meta_args = match MetaArgs::try_from(args_str.clone()) {
+        Ok(args) => args,
+        // Exit on error
+        Err(error) => {
+            log_error!(context,
+                "zome: invoke_emit_signal failed to \
```

Comment:
This error text looks wrong.. 
```suggestion
                "zome: invoke_meta failed to \
```